crates/libcgroups/src/systemd/manager.rs

```diff
@@ -74,12 +107,11 @@ impl Manager {
             container_name,
             unit_name: Self::get_unit_name(&destructured_path),
             destructured_path,
-            client: Client::new().context("failed to create dbus client")?,
+            client,
         })
     }
 
-    fn destructure_cgroups_path(cgroups_path: PathBuf) -> Result<CgroupsPath> {
-        log::debug!("CGROUPS PATH IS {:?}", cgroups_path);
+    fn destructure_cgroups_path(cgroups_path: &Path) -> Result<CgroupsPath> {
```

Comment:
Maybe this should be implemented `impl TryFrom<Path> for CgroupPath`.  What do you think?